provision/provision.go

```diff
@@ -0,0 +1,162 @@
+// This package will fetch the recommendation from the recommendation Queue and provision the scale in/out
+// based on command.
+package provision
+
+import (
+	"scaling_manager/cluster"
+	"time"
+)
+
+var counter uint8 = 1
+
+// This struct contains the State of the opensearch scaling manager
+// States can be following:
```

Comment:
We need to add another state so that we can know that we can provision the recommendation.